modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,130 @@
+= Audit Logging
+:description: Learn how to use Redpanda's audit logging capabilities.
+
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
+
+Redpanda's auditing capabilities support recording both administrative and operational interactions with topics and with users. Redpanda complies with the Open Cybersecurity Schema Framework (OCSF), providing a predictable and extensible solution that works seamlessly with industry standard tools.
+
+NOTE: All audit logging is configured at the cluster level. Redpanda supports excluding specific topics or principals from audit to help reduce noise in the log. Audit logging is disabled by default.
+
+== Audit log flow
+
+The Redpanda audit log mechanism functions similar to the Kafka flow you may be familiar with. When a user interacts with another user or with a topics, Redpanda writes an event to a specialized audit topic. The audit topic is immutable. Only Redpanda itself can write to it -- users are prevented from writing to the topic directly and the Kafka API cannot create or delete it.
+
+image:shared:audit-loging-flow.png[Audit log flow]
+
+By default, any management and authentication actions performed on the cluster yield messages written to the audit log topic that are retained for seven days. Interactions with all topics by all principals are audited. Actions performed using the Kafka API and Admin API are all audited, as are actions performed directly through rpk.
+
+Messages recorded to the audit log topic comply with the https://schema.ocsf.io/[open cybersecurity schema framework]. Any number of analytics frameworks, such as Splunk or Sumo Logic, can receive and process these messages. Using an open standard ensures Redpanda's audit logs coexist with those produced by other IT assets, powering holistic monitoring and analysis of your assets.
+
+== Audit log configuration options
+
+Redpanda's audit logging mechanism supports several options to control the volume and availability of audit records. Configuration is applied at the cluster level using the standard https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-property-configuration/[cluster configuration mechanism]. The cluster configuration options exclusive to audit logging include the following:
+
```

Comment:
Did you consider placing this content in a table? Might be easier to read. 
Headings could be: Property | Description | Default
For each, you could link the name to the actual property in the Reference.